spec/pantry_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+RSpec.describe Kitchen::Pantry do
+  let(:instance) { described_class.new }
+
+  it 'creates a hash' do
+    expect(instance).to be_truthy
+  end
+
+  context 'when the pantry has contents' do
+    before { instance.store 'bar', label: 'foo' }
+
+    it 'adds an item to the pantry with the provided label' do
+      expect(instance).to include(:foo)
+    end
+
+    it 'gets an item from the pantry' do
+      expect(instance.get(:foo)).to eq('bar')
+    end
+
+    it 'raises if requested item is not in pantry' do
+      expect{ instance.get!('thud') }.to raise_error(
+        Kitchen::RecipeError, "There is no pantry item labeled 'thud'"
+      )
+    end
```

Comment:
Would be good to also test the behavior of `get` (not `get!`) when the item isn't present -- it should return `nil`

So you might break it down like 

```ruby
context '#get' do
  it 'returns a present item with either a string or symbol label' do
    expect(instance.get(:foo)).to eq('bar')
    expect(instance.get('foo')).to eq('bar')
  end

  it 'returns nil when an item is not present' do
    expect(instance.get(:blah)).to be_nil
  end
end

context '#get!' do
  it 'returns a present item with either a string or symbol label' do
    expect(instance.get!(:foo)).to eq('bar')
    expect(instance.get!('foo')).to eq('bar')
  end

  it 'raises if the item is not present' do
    # your existing raise spec here
  end
end
```

Reply:
Good point about having positive and negative path tests for both. I've incorporated this and reworked the layout to match 👍